terraform/dev/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "app" {}
```

Comment:
Does "dev" stand for the "dev" environment?  This looks like an auto scaling group module more than the environment instantiation.

Reply:
Yes. My intent is to separate the basic configs by env for now, since we're just starting to maintain infra definitions with terraform. If I am able to use the ASG module you mention, I will probably end up restructuring this a bit.